cockatrice/src/dlg_settings.cpp

```diff
@@ -486,25 +487,90 @@ void UserInterfaceSettingsPage::retranslateUi()
     tapAnimationCheckBox.setText(tr("&Tap/untap animation"));
 }
 
-
 DeckEditorSettingsPage::DeckEditorSettingsPage()
 {
-    QGridLayout *generalGrid = new QGridLayout;
-    
-    generalGrid->addWidget(new QLabel(tr("Nothing is here... yet")), 0, 0);
-    
-    generalGroupBox = new QGroupBox;
-    generalGroupBox->setLayout(generalGrid);
-    
-    QVBoxLayout *mainLayout = new QVBoxLayout;
-    mainLayout->addWidget(generalGroupBox);
-    
-    setLayout(mainLayout);
+    auto *lpGeneralGrid = new QGridLayout;
+    auto *lpSpoilerGrid = new QGridLayout;
```

Comment:
All `GeneralGrid` could be removed maybe?
It has been empty for a long time now, and most likely will for some more... I don't know of any feature in the planning that could end up here!

And it looks not really nice 😄 
![settings](https://user-images.githubusercontent.com/9874850/34703367-e0f5d79e-f4f2-11e7-9a09-fb8b99f1aea5.png)

Reply:
I don't see harm in leaving it - let's not increase the amount of stuff being changed since this is already large